.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "env": {
```

Comment:
Really neat! We should probably add the environment variable NODE_ENV "development" to make sure that the correct config is going to be applied at run time.